### ISSUE TITLE
Request for the Training & Test Set PDB ID Lists

### ISSUE THREAD

**Issue by wlin16** [RosettaCommons/RFantibody#42]
Hello RFantibody maintainers,

Thank you for releasing this great work! Could you please provide the exact PDB ID lists (or downloadable CSV/TXT files) for the three antibody‐related datasets used during fine-tuning:
	1.	SAbDab antibody–antigen complexes
	2.	TCR–pMHC complexes
	3.	Loop-mediated hetero-dimeric interfaces from PDB

Thank you!
